deploy/team-api-update.js

```diff
@@ -21,11 +34,11 @@ function spawn(action_description, path, args) {
   });
 }
 
-function bundleInstall() {
+TeamApiUpdater.prototype.bundleInstall = function bundleInstall() {
```

Comment:
Yep, here's a nice summary with examples:

http://www.therightcode.net/naming-functions-yes-but-not-all-the-time/